### PR TITLE
Dev environment updates

### PR DESCRIPTION
**Pull request by beejunk**
I was looking to help with Shin KGS, and ran into  issues getting the local development environment working. This PR adds the changes I made to get these issues resolved. These changes are:

* Updated node-sass and react-scripts to their latest versions. The older versions would not run in an Ubuntu 20.04 environment.
* Updated the main CSS import to point directly to the SCSS file. react-scripts will automatically build SCSS files if node-sass is present, so there's no need to do that manually.
* Added a Vercel (used to be now.sh) serverless-function to proxy KGS requests. This allows the Vercel preview deployments to work. Since Vercel has removed full-stack Node support in favor of a serverless architecture, the original proxy server setup no longer works. You can see this when visiting https://shinkgs-master.now.sh/ - the proxy requests are now returning 404 errors.
* Switch the local dev proxy to run through the Webpack dev server. This allows proxying to the KGS server without using HTTPS (and without needing to add a certificate exception in your browser).
* Update README accordingly.

### CODE REVIEW

**Comment by beejunk** [comment]
Whoops, I accidentally committed a `package-lock.json` file! I've amended the commit to remove to remove that file.

Vercel deploy of these changes can be seen here: https://shinkgs.ectopod.vercel.app

File: README.md [hunk 1]
```diff
@@ -91,22 +92,25 @@ We use Travis as a continuous integration service. The [Travis job](.travis.yml)
...
+If you are developing the app and want to deploy in the cloud, use the [Vercel service](https://www.vercel.com):
 
 ```
-yarn global add now
-yarn now-deploy
+# Install `vercel` globally
+yarn global add vercel 
+
+# Deploy the project.
+vercel
```
Comment (leeschumacher) [inline README.md:110]: this part didn't work for me.  yarn add said it worked, but i can't find an executable called 'vercel' anywhere.

File: README.md [hunk 2]
```diff
@@ -110,7 +110,17 @@ yarn global add vercel
...
 ```
 
-This builds the app then pushes it out to the cloud.
+This builds the app then pushes it out to the cloud. If you have not used Vercel before, you will be prompted to log-in using your email before deploying.
+
+## Troubleshooting
+
+### Installing Node.js on Windows
+
+It is highly recommended that you install the [Chocalety package manager](https://chocolatey.org/) before installing Node.js on Windows. After Cholatey is installed, make sure to check the 'Automatically install the necessary tools' checkbox when prompted during the Node.js installation. These tools are needed to build some ShinKGS dependencies.
```
Comment (leeschumacher) [inline README.md:119]: i see 3 different spellings of "chocolatey" in this paragraph.  I'm hoping at least one of them is right?

File: README.md (hunk 1 above)
Comment (beejunk) [reply README.md:110]: I've added a 'Troubleshooting' section that includes some details on checking that the Yarn 'bin' directory is in the path environment variable.

File: README.md (hunk 2 above)
Comment (beejunk) [reply README.md]: Yikes! Okay, I've updated all spellings to match what is in the URL (which is correct only because I copied the link 😀)

**Pull request merged**